src/components/horizontal-term-line/HorizontalTermLine.tsx

```diff
@@ -30,7 +33,13 @@ const HorizontalTermLine: React.FC<HorizontalTermLineProps> = ({
   }
 
   return (
-    <div data-cy={dataCy} className="text-small-caption horizontal-term-line">
+    <div
+      data-cy={dataCy}
+      className={clsx(
+        "text-small-caption horizontal-term-line",
+        classNames || ""
```

Comment:
Make the classNames param optional if its not already

```suggestion
        classNames
```